src/HealthChecks.AzureServiceBus/DependencyInjection/AzureServiceBusHealthCheckBuilderExtensions.cs

```diff
@@ -244,15 +245,17 @@ public static class AzureServiceBusHealthCheckBuilderExtensions
         string? name = default,
         HealthStatus? failureStatus = default,
         IEnumerable<string>? tags = default,
-        TimeSpan? timeout = default) =>
+        TimeSpan? timeout = default,
+        bool isExceptionDetailsRequired = true) =>
```

Comment:
I would not touch any of extension methods here to add one more argument. The appointment of an option class (for any health check) is to facilitate the support and configuration of health checks. This method (as well as some others) contains `Action<AzureServiceBusQueueHealthCheckOptions>? configure` delegate, so just set the value of desired setting inside it.